src/components/Notifications.tsx

```diff
@@ -1,56 +1,47 @@
-import { useMemo, useState } from "react";
-import useOnclickOutside from "react-cool-onclickoutside";
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+import { beamerInitialize } from "react-beamer";
+import { useEffect } from "react";
+import { User } from "../model";
+import { defaultLanguage } from "./i18n/MenuIntlProvider";
+import { useAppConfiguration } from "../AppConfiguration";
 
-interface NotificationsProps {
-  notifications: ReadonlyArray<string>;
-  emptyNotificationText: string;
-}
+// this is a temporary beamer account with limited functions
+export const DEFAULT_BEAMER_ID = "wCWRrvwa50706";
+export const BEAMER_URL = null;
 
-export const Notifications = ({
-  notifications,
-  emptyNotificationText,
-}: NotificationsProps) => {
-  const count = notifications.length;
-  const [openNotification, setOpenNotification] = useState(false);
-  const notificationsRef = useOnclickOutside(() => setOpenNotification(false));
+export const BEAMER_CONFIG = {
+  selector: ".beamer-icon",
+  top: -10,
+  right: -10,
+  language: defaultLanguage,
```

Comment:
You should use the selected language, right?

Reply:
yes, it's right ... I use it to initialize beamer ... in this part: 
```
 beamerInitialize(beamerId, BEAMER_URL, {
      ...BEAMER_CONFIG,
      //---------------Visitor Information---------------
      user_firstname: user.fullname,
      user_email: user.email,
      language: user.lang,
    });
```